lib/logstash/inputs/jdbc.rb

```diff
@@ -160,6 +160,17 @@ class LogStash::Inputs::Jdbc < LogStash::Inputs::Base
   # Whether to force the lowercasing of identifier fields
   config :lowercase_column_names, :validate => :boolean, :default => true
 
+  # Enable the usage of encoding conversions, if not enabled no conversion
+  # will be performed. (default => false).
+  # IF enabled string will be converted uniformly to UTF-8 from their source encoding, this
+  # is handy in some situations to avoid problems when non UTF-8 encoded strings are used.
+  config :enable_encoding, :validate => :boolean, :default => false
+
+  # IF enable_encoding is true (default=false) it let's you select which string columns
+  # should be converted to UTF-8. If empty it will convert all columns. To select
+  # a column to be converted, add it's name to this list.
+  config :columns_to_encode, :validate => :array, :default => []
+
```

Comment:
I don't understand where the source charset is specified? to properly perform transcoding, we need to be able to specify the source charset per column.